global.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module 'matched';
+declare module 'watch-dependency-graph';
```

Comment:
Oooof, yeah gotta do this one too.

Reply:
I saw the issue about exchanging `matched` for `tiny-glob`. Maybe we should to that before working out the types for matched if we're gonna do it anyways.